tasks/sep/fp-recovery/004-recover-bonds/justfile

```diff
@@ -0,0 +1,18 @@
+set positional-arguments
+
+# default recipe to display help information
+default:
+  @just --list
+
+# Generate the `input.json` with a DelayedWETH address to recover bonds
+generate-input delayed_weth:
+  #!/usr/bin/env bash
+  AMOUNT=$(cast --to-hex $(jq add ./meta.json))
```

Comment:
sanity check -- what size ints does `jq add` use? Could this cease to work if we're using much larger ints, say full fixed-point 1e18 WEI?